drivers/src/memory_layout.rs

```diff
@@ -38,12 +38,17 @@ pub const PCR_LOG_ORG: u32 = 0x50004800;
 pub const MEASUREMENT_LOG_ORG: u32 = 0x50004C00;
 pub const FUSE_LOG_ORG: u32 = 0x50005000;
 pub const DPE_ORG: u32 = 0x50005400;
-pub const PCR_RESET_COUNTER_ORG: u32 = 0x50006400;
-pub const DATA_ORG: u32 = 0x50006800;
-pub const STACK_ORG: u32 = 0x5001A000;
+pub const PCR_RESET_COUNTER_ORG: u32 = DPE_ORG + DPE_SIZE;
+pub const DATA_ORG: u32 = PCR_RESET_COUNTER_ORG + PCR_RESET_COUNTER_SIZE;
+
+pub const STACK_ORG: u32 = DATA_ORG + DATA_SIZE;
 pub const ROM_STACK_ORG: u32 = 0x5001C000;
-pub const ESTACK_ORG: u32 = 0x5001F800;
-pub const NSTACK_ORG: u32 = 0x5001FC00;
+
+pub const ESTACK_ORG: u32 = STACK_ORG + STACK_SIZE;
+pub const ROM_ESTACK_ORG: u32 = 0x5001F800;
+
+pub const NSTACK_ORG: u32 = ESTACK_ORG + ESTACK_SIZE;
+pub const ROM_NSTACK_ORG: u32 = 0x5001FC00;
```

Comment:
Because of RT hitless update, it makes me a little nervous to compute these offsets dynamically. I think we may need to be careful about which things get moved across hitless update.

Do we need a concept of an RT that makes a breaking change to the memory model and can't be hitlessly update to.